src/test/java/com/wcc/platform/controller/AboutControllerTest.java

```diff
@@ -121,4 +123,50 @@ void testCollaboratorOKResponse() throws Exception {
                         .andExpect(jsonPath("$.collaborators[0].network[0].type", is("LINKEDIN")))
                         .andExpect(jsonPath("$.collaborators[0].network[0].link", is("collaborator_link")));
     }
+
+    @Test
+    void testCodeOfConductNotFound() throws Exception {
+        when(service.getCodeOfConduct()).thenThrow(new ContentNotFoundException("Not Found Exception"));
+
+        mockMvc.perform(get("/api/cms/v1/code-of-conduct")
```

Comment:
you could extract to a constant this string and reuse in the test.

```suggestion
      private static final String API = "/api/cms/v1/code-of-conduct";
      
      .... 
      
      
      mockMvc.perform(get(API)
```